### DISCUSSION TITLE
How to take backup of data?

### DISCUSSION THREAD

**Discussion by manigandan-dorairaj** [alainm23/planify#834] [Q&A]
Hi,

New to the Planner app. Really well designed and intuitive. I have been trying to figure out how to take backup of data. I think the data is stored locally and I don't want to lose it in case my system fails at sometime.

Thanks in advance.

**Comment by alainm23** [comment]
An import/export and automatic backup system is being developed to solve this problem.

**Comment by alainm23** [comment]
This functionality is now available in Planify 4.3.
